GoogleForCreators/web-stories-wp
# Design Panel: Add global/local colors to Color Picker

## Context
Adds global and local colors to the color picker. Uses `AdvancedDropDown` component for changing between the current/global color types since we don't have a good alternative for the new design yet.


## Summary



## Relevant Technical Choices



## To-do

- [x] Fix focus not being visible when using arrow keys



## User-facing changes
Local & global colors are now in the color picker, too.


## Testing Instructions



### QA






This PR can be tested by following these steps:


1. Add a shape and change its color.
2. Open the color picker
3. Add color by clicking on the `+` icon in the color picker.
4. Switch to Global color from the select
5. Add a color there as well.
6. Verify that the color appeared in the design panel under both local and global colors.
7. Add a text element
8. Open the color picker
9. Verify that both local and global colors appear for the text as well and can be applied by clicking on it.
10. Test the same for Page background as well.

NOTE: The dropdown used in the color picker is using old design. This should be changed together with redesign.

### UAT







This PR can be tested by following these steps:

1.

## Reviews

### Does this PR have a security-related impact?



### Does this PR change what data or activity we track or use?



### Does this PR have a legal-related impact?



## Checklist




---



Fixes #3634
